src/utils/joinWaitListGoogleFormScript.gs

```diff
@@ -28,7 +27,11 @@ function onFormSubmit(e) {
     muteHttpExceptions: true,
   }
 
-  const response = UrlFetchApp.fetch("[deployed-link]/join-wait-list", options)
+  // use actual link for the production backend
+  const response = UrlFetchApp.fetch(
+    "https://trainee-app-backend-staging-1593.up.railway.app/join-wait-list",
```

Comment:
please avoid hard coding urls in the project. This might break if the backend deployment url changes

Reply:
Use the api base url from the env file